### ISSUE TITLE
Select parameters for oversegmentation and classifier by optimising validation set error

### ISSUE THREAD

**Issue by jsherrah** [RockStarCoders/alienMarkovNetworks#27]
trainClassifier.py already does the grid search for classifier parameters.  Need a script that searches for the 2 superPixel (SLIC) parameters outside of this.
1. for P1,P2 (superPixel params) in range of combos:
   1. createFeatures.py to generate feature set for training set.
   2. trainClassifier.py to grid search for best classifier given these features.
   3. run classifyAllImages.sh to generate class prediction images for validation set.
   4. run evalPredictions.py on classifier results to get validation set error.
2. Select optimal P1,P2 to give minimum validation set error.
3. This gives us the best classifier for the training (and validation) data.
